src/testCombiner.ts

```diff
@@ -90,7 +90,20 @@ export function genTestClass(
   const classNameModified = className.replace(/\$/g, '_');
   // Merge all tests' classAnnotation arrays into a single Set
   const classAnnotations = new Set(([] as string[]).concat(...tests.map((test) => test.classAnnotations)));
-  const uniqueTests = tests[0].id ? deduplicateTestNames(tests) : tests;
+  // Standardise test interfaces
+  const standardisedTests = tests.map((t) => {
+    if (!t.body) {
+      t.body = t.testBody || t.test;
+      delete t.testBody;
+      delete t.test;
+    }
+    if (!t.name) {
+      t.name = t.testName;
+      delete t.testName;
+    }
+    return t;
+  });
+  const uniqueTests = tests[0].id ? deduplicateTestNames(standardisedTests) : tests;
```

Comment:
This should have been:

```suggestion
  const uniqueTests = tests[0].id ? deduplicateTestNames(standardisedTests) : standardisedTests;
```

but should now be:

```suggestion
  const uniqueTests = tests[0].id ? deduplicateTestNames(tests) : tests;
```